### ISSUE TITLE
Sails 0.11.0 Custom 403 Message

### ISSUE THREAD

**Issue by crh3675** [balderdashy/sails#2796]
It seems the default template for 403 expects a variable called "error".  However, the responses/forbidden sends  values to the template in an object with key called "data".

What this means is, I had to change to the default behavior of the 403 template to use:

```
          <% if(typeof data !== 'undefined' && data.error ) { %>
          <%= data.error %>          
          <% } else if (typeof error !== 'undefined') { %>
          <%= error %>
          <% } else { %>
          You don't have permission to see the page you're trying to reach.
          <% } %>
```

Not a big issue but just wanted to make note that in order to override the default message for csrf errors, some adjustments are needed, both in responses/forbidden and the view template:

```
module.exports = function forbidden (data, options) {

  // Get access to `req`, `res`, & `sails`
  var req = this.req;
  var res = this.res;
  var sails = req._sails;
  var csrfError = ( data == 'CSRF mismatch' ); // Catch csrf mismatch

  // Set status code
  res.status(403);

  // Log error to console
  if (data !== undefined) {
    sails.log.verbose('Sending 403 ("Forbidden") response: \n',data);
  }
  else sails.log.verbose('Sending 403 ("Forbidden") response');

  // Only include errors in response if application environment
  // is not set to 'production'.  In production, we shouldn't
  // send back any identifying information about errors.
  if (sails.config.environment === 'production') {
    data = undefined;
  }
  
// Customize message for csrf
  if(csrfError) {      
     data = { error : 'The form you attempted to submit has expired' };     
  }
```

**Comment by sailsbot** [comment]
Thanks for posting, @crh3675.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
